assets/css/twenty-seventeen.scss

```diff
@@ -102,7 +102,17 @@
 		color: #fff;
 	}
 }
-
+.colors-dark {
+	.woocommerce-pagination {
+		span.page-numbers.current {
+			background-color: #fff;
+		}
+		.next.page-numbers,.prev.page-numbers,a.page-numbers,span.page-numbers{
+			background: #505050;
+		}	
+		
+	}
+}
```

Comment:
- This doesn't keep the color fading and switching effect that happens when you hover over a button. Can you make the number buttons have the same effect as the arrow buttons when hovered?
- This code should go in the dark colors style section starting at line 1008. That way all of the dark colors style is in the same place.